crates/red_knot_python_semantic/src/symbol.rs

```diff
@@ -355,9 +356,15 @@ fn core_module_scope(db: &dyn Db, core_module: KnownModule) -> Option<ScopeId<'_
 /// The type will be a union if there are multiple bindings with different types.
 pub(super) fn symbol_from_bindings<'db>(
     db: &'db dyn Db,
+    scope: ScopeId<'db>,
     bindings_with_constraints: BindingWithConstraintsIterator<'_, 'db>,
 ) -> Symbol<'db> {
-    symbol_from_bindings_impl(db, bindings_with_constraints, RequiresExplicitReExport::No)
+    symbol_from_bindings_impl(
+        db,
+        bindings_with_constraints,
+        RequiresExplicitReExport::No,
+        scope,
```

Comment:
Nit: Can we align the argument ordering. `symbol_from_bindings` takes `scope` as second argument before `bindings` and `symbol_from_bindings_impl` takes `scope` as the last argument.